membership-token/js/src/accounts/Store.ts

```diff
@@ -102,17 +107,17 @@ export class StoreAccountData {
   }
 }
 
-const storeAccountDataStruct = new beet.BeetStruct<
+const storeAccountDataStruct = new beet.FixableBeetStruct<
   StoreAccountData,
   StoreAccountDataArgs & {
     accountDiscriminator: number[];
   }
 >(
   [
-    ['accountDiscriminator', beet.fixedSizeArray(beet.u8, 8)],
+    ['accountDiscriminator', beet.uniformFixedSizeArray(beet.u8, 8)],
     ['admin', beetSolana.publicKey],
-    ['name', beet.fixedSizeUtf8String(NAME_MAX_LEN)],
-    ['description', beet.fixedSizeUtf8String(DESCRIPTION_MAX_LEN)],
+    ['name', beet.utf8String],
```

Comment:
It looks like having this you are able to get rid of checkByteSizes in [createStore.ts](https://github.com/metaplex-foundation/metaplex-program-library/blob/master/membership-token/js/src/instructions/createStore.ts) and [createMarket.ts](https://github.com/metaplex-foundation/metaplex-program-library/blob/master/membership-token/js/src/instructions/createMarket.ts) 

```typescript
  const name = checkByteSizes(args['name'], NAME_MAX_LEN);
  const description = checkByteSizes(args['description'], DESCRIPTION_MAX_LEN);
```

Reply:
type has been changed after api generation, should we keep fixed size manually or we able to move forward with utf8string type

Reply:
Please check it by adding one more test for createStore.ts with a short store name & description. If the local validator allows it, then it's fine.

Reply:
added